rfcs/rfc-composite-pdfs.md

```diff
@@ -0,0 +1,154 @@
+- Start Date: 2021-09-08
+- Target Major Version: 
+- Reference Issues: n/a
+- Entity Issue: https://github.com/bcgov/entity/issues/7217
+- Implementation PR: 
+
+
+# Summary
+
+When a user is filing a Cooperative Incorporation, Rules and Memorandum documents uploaded by the user need to be stamped with the current registrar's signature and texts such as date and time of filing and incorporation number. These documents are PDF files which need to be composited with image and text to generate certified copies of the user documents.
```

Comment:
_incorporation number_ isn't known until the Filer creates the company.
so it seems like this would be a service performed in the Filer?

Reply:
Yes. Added that to the RFC. Thanks!